src/status_im/utils/universal_links/core.cljs

```diff
@@ -158,6 +162,16 @@
   (log/info :store-url-for-later)
   {:db (assoc db :universal-links/url url)})
 
+(rf/defn handling
+  {:events [:universal-links/remove-handling]}
```

Comment:
event name is the same as for remove-handling

Reply:
fixed, thanks